src/common/components/repositories-list/index.js

```diff
@@ -40,18 +48,12 @@ class RepositoriesList extends Component {
     }
   }
 
-  renderRepository(repo) {
-    const { fullName } = repo;
-    const status = this.props.repositoriesStatus[fullName] || {};
-
+  renderRow(snap) {
+    const { fullName } = parseGitHubRepoUrl(snap.git_repository_url);
     return (
       <RepositoryRow
-        key={ `repo_${repo.fullName}` }
```

Comment:
It's a fullName of repository rather then a snap, but - whatever, it's internal React key anyway.